### PR TITLE
:new: docker-sphinx

### PR DESCRIPTION
**Pull request by tkoyama010**
Subject: :new: docker-sphinx

### Feature or Bugfix
- Feature

### Purpose
- Adding a section docker-sphinx in installation document.

### Detail
- None

### Relates
- None

### CODE REVIEW

File: doc/usage/installation.rst [hunk 1]
```diff
@@ -187,3 +187,46 @@ You can also download a snapshot of the Git repo in either `tar.gz`__ or
 __ https://github.com/sphinx-doc/sphinx
 __ https://github.com/sphinx-doc/sphinx/archive/master.tar.gz
 __ https://github.com/sphinx-doc/sphinx/archive/master.zip
+
+
+docker-sphinx
+-------------
+
+docker-sphinx is Docker images for Sphinx.
```
Comment (tkoyama010) [inline doc/usage/installation.rst]: It is better to add URL.
```suggestion
`docker-sphinx <https://hub.docker.com/r/sphinxdoc/sphinx>`_ is Docker images for Sphinx.
```

**Comment by tkoyama010** [comment]
That's all it is ready for review and merge.

File: doc/usage/installation.rst [hunk 2]
```diff
@@ -187,3 +187,45 @@ You can also download a snapshot of the Git repo in either `tar.gz`__ or
 __ https://github.com/sphinx-doc/sphinx
 __ https://github.com/sphinx-doc/sphinx/archive/master.tar.gz
 __ https://github.com/sphinx-doc/sphinx/archive/master.zip
+
+
+docker-sphinx
```
Comment (tk0miya) [inline doc/usage/installation.rst]: Docker is a kind of software platform. So it is better to use "Docker" simply to me.

File: doc/usage/installation.rst [hunk 3]
```diff
@@ -187,3 +187,45 @@ You can also download a snapshot of the Git repo in either `tar.gz`__ or
...
+-------------
+
+`docker-sphinx <https://hub.docker.com/r/sphinxdoc/sphinx>`_ is Docker images for Sphinx.
```
Comment (tk0miya) [inline doc/usage/installation.rst]: How about this instead?

Docker images for Sphinx are published on the [Docker Hub](https://hub.docker.com/). There are two kind of images:

* [sphinxdoc/sphinx](https://hub.docker.com/repository/docker/sphinxdoc/sphinx)
* [sphinxdoc/sphinx-latexpdf](https://hub.docker.com/repository/docker/sphinxdoc/sphinx-latexpdf)

Former one is used for standard usage of Sphinx, and latter one is mainly used for PDF builds using LaTeX. Please choose one for your purpose.

.. note::

   sphinxdoc/sphinx-latexpdf contains TeXLive packages. So it is very large (over 2GB!).

File: doc/usage/installation.rst [hunk 4]
```diff
@@ -187,3 +187,45 @@ You can also download a snapshot of the Git repo in either `tar.gz`__ or
...
+It includes images
+
+- sphinxdoc/sphinx
+- sphinxdoc/sphinx-latexpdf
+
+.. note::
+
+   sphinxdoc/sphinx-latexpdf contains TeXLive packages. So the image is very large (over 2GB!).
+
+You can create a Sphinx project::
```
Comment (tk0miya) [inline doc/usage/installation.rst:209]: The description after this line is not about "installation". So it might be better to move other place.

**Review by tk0miya — CHANGES_REQUESTED** [review]
Thank you for working. LGTM with nits

File: doc/usage/installation.rst (hunk 2 above)
Comment (tkoyama010) [reply doc/usage/installation.rst]: GOTCHA . I changed the title.

File: doc/usage/installation.rst (hunk 3 above)
Comment (tkoyama010) [reply doc/usage/installation.rst]: LGTM

File: doc/usage/installation.rst (hunk 4 above)
Comment (tkoyama010) [reply doc/usage/installation.rst:209]: You're right. I added TODO as a tentative solution.

**Comment by tkoyama010** [comment]
Thank @tk0miya for your review 👍

**Comment by tk0miya** [comment]
Thanks!

**Pull request merged**